<ui version="4.0" >
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="ImIcon.qrc" >
    <normaloff>:/pwytter</normaloff>:/pwytter</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QLabel" name="lblName" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPic" >
    <property name="geometry" >
     <rect>
      <x>350</x>
      <y>10</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMsgSend" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>260</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTextEdit" name="ShowChat" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>50</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="lineWrapMode" >
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="ImIcon.qrc" />
 </resources>
 <connections/>
</ui>
